mapstore/localConfig.json

```diff
@@ -0,0 +1,606 @@
+                "autoload": true
+              },
+              "sadre": {
+                "url": "http://www.sandre.eaufrance.fr/atlas/srv/fre/csw",
+                "type": "csw",
+                "title": "le catalogue du Sandre",
+                "autoload": true
+              },
+              "webservice_energy": {
+                "url": "http://geocatalog.webservice-energy.org/geonetwork/srv/fre/csw",
+                "type": "csw",
+                "title": "le catalogue de webservice-energy",
+                "autoload": true
+              },
+              "ifremer": {
+                "NOTE": "NOT WORKING, requires login. This is configured on the current viewer but do not provide any response",
+                "url": "http://www.ifremer.fr/geonetwork/srv/fre/csw",
+                "type": "csw",
+                "title": "le catalogue de l'Ifremer",
+                "autoload": true
+              },
+              "geograndest": {
+                "url": "https://www.geograndest.fr/geonetwork/srv/fre/csw",
+                "type": "csw",
+                "title": "le catalogue de GéoGrandEst",
+                "autoload": true
+              }
+            },
+            "staticServices": {
+              "default_map_backgrounds": {
+                "type": "backgrounds",
+                "title": "defaultMapBackgroundsServiceTitle",
+                "autoload": true,
+                "backgrounds": [{
+                    "type": "osm",
+                    "title": "Open Street Map",
+                    "name": "mapnik",
+                    "source": "osm",
+                    "group": "background"
+                }, {
+                    "type": "tileprovider",
+                    "title": "NASAGIBS Night 2012",
+                    "provider": "NASAGIBS.ViirsEarthAtNight2012",
+                    "name": "Night2012",
+                    "source": "nasagibs",
+                    "group": "background"
+                }, {
+                    "type": "tileprovider",
+                    "title": "OpenTopoMap",
+                    "provider": "OpenTopoMap",
+                    "name": "OpenTopoMap",
+                    "source": "OpenTopoMap",
+                    "group": "background"
+                  },
+                  {
+                    "source": "ol",
+                    "group": "background",
+                    "title": "Empty Background",
+                    "fixed": true,
+                    "type": "empty"
+                }]
+              }
+            }
+          }
+        },
+        "locales": {
+          "supportedLocales": {
+            "en": {
+              "code": "en-US",
+              "description": "English"
+            },
+            "it": {
+              "code": "it-IT",
+              "description": "Italiano"
+            },
+            "fr": {
+              "code": "fr-FR",
+              "description": "Français"
```

Comment:
For my sake, do you support german & spanish too ?
If yes, can it be added ?

Reply:
@fvanderbiest yes, we support also those two, no problem in adding them back.
Just a notice: this configuration has been created to fullfill the georchestra project requirements, it is not the standard mapstore configuration
If you want to change it / improve it, no problem from us